exonum/src/encoding/float.rs

```diff
@@ -369,12 +367,10 @@ mod tests {
     #[test]
     #[allow(dead_code)]
     fn test_f64_struct() {
-        encoding_struct!(
-            struct Msg {
-                single_float: F64,
-                vec: Vec<F64>,
-            }
-        );
+        encoding_struct!(struct Msg {
```

Comment:
Ouch, for what reason they started to format code inside macros? Looks really awful

Reply:
Personally, I like the idea. :upside_down_face: